src/Components/Todo/Todo.js

```diff
@@ -0,0 +1,51 @@
+import '../Style/Style.css';
+
+const MyTodo = () => {
+  const data = [
+    {
+      id: 1,
+      title: 'Setup development environment',
+      author: ' Okpara Blessing',
+    },
+    {
+      id: 2,
+      title: 'Develop website and add content',
+      author: ' Okpara',
+    },
+    {
+      id: 3,
+      title: 'Deploy to live server',
+      author: ' Blessing',
+    },
+  ];
+  return (
+    <>
+
+      <div className="container">
+        {data.map((item) => (
+          <div className="eachBook" key={item.id}>
+            <h2>{item.title}</h2>
+            <p>
+              By
+              {item.author}
+            </p>
+            <button type="button" id="remove">Remove</button>
+          </div>
+        ))}
+        <form>
+          <hr />
+          <h3>Add New Book</h3>
+          <div className="formInput">
+            <input type="text" id="title" placeholder="Book Title" />
+            <br />
+            <input type="text" id="name" placeholder="Author" />
+            <br />
+            <button type="submit">ADD BOOK</button>
+          </div>
+        </form>
+      </div>
```

Comment:
- [ ] Please make sure that breaks this component into the following components: :pray:
  * `<BookList>`
    * Renders a list of books using the `<Book>` component
  * `<Book>`
    * Takes the book data as a prop and renders the title and the author
    * Has a button for removing books
      * the button wouldn't remove the books *yet*, this is just a placeholder for now
  * `<NewBookForm>`
    * Renders a form with inputs for the title and the author of the book
    * Has a button to submit
      * the button wouldn't add the book *yet*, this is just a placeholder for now